libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationViewModel.java

```diff
@@ -212,6 +218,15 @@ public MapboxNavigation getNavigation() {
     return navigation;
   }
 
+  /**
+   * Sets the callback for reroute events and faster routes
+   *
+   * @param callback to set
+   */
+  public void addCallback(Callback callback) {
```

Comment:
Is there any possibility in which we could end up adding the same callback more than once? If so, we should check it.

Noting that if that's the case, if we'd use a `Set` (`CopyOnWriteArraySet`) instead of a `List` (`CopyOnWriteArrayList`) this check wouldn't be necessary (it's checked internally).

👀 [`Set<E>`](https://docs.oracle.com/javase/8/docs/api/java/util/Set.html) and [`List<E>`](https://docs.oracle.com/javase/8/docs/api/java/util/List.html)